### DISCUSSION TITLE
New OS?

### DISCUSSION THREAD

**Discussion by KittyForest** [Sugaryyyy/FemboyOS#4] [General]
So is this supposed to become a new operating system like windows/linux/mac?

**Comment by vel2006** [comment]
> So is this supposed to become a new operating system like windows/linux/mac?

In short, I would assume so. In long, Linux is not an os, but rather a kernel. But yes, it is a flavor/distro of Linux which is compiled into an OS. And is used the same way that another Linux flavor/distro would be used, but this one does not seem to be as developed as others ie: Ubuntu, Debian, Arch, Chromium, Gentoo, Alpine, and many many others, but more instead as a passion project.

**Comment by ghost** [comment]
I want to use FemboyOS
